include/benchmark/benchmark_api.h

```diff
@@ -709,8 +709,12 @@ class Fixture: public internal::Benchmark {
       this->TearDown(st);
     }
 
+    // These will be deprecated ...
     virtual void SetUp(const State&) {}
     virtual void TearDown(const State&) {}
+    // ... In favor of these.
+    virtual void SetUp(State& st) { SetUp(static_cast<const State&>(st)); }
```

Comment:
gah, completely read it wrong. i thought it was the other way around (with the old ones forwarding to the new ones)